Games/types/Mafia/roles/cards/ActWhileDead.js

```diff
@@ -0,0 +1,14 @@
+const Card = require("../../Card");
+
+module.exports = class ActWhileDead extends Card {
+  constructor(role) {
+    super(role);
+
+    this.meetingMods = {
+        "*": {

```

Comment:
need to exclude village mafia cult

Reply:
Not sure how to do this. In an earlier draft we had:
```
        shouldMeet: function (meetingName) {
          if (
            meetingName == "Village" ||
            meetingName == "Mafia" ||
            meetingName == "Cult" ||
            meetingName == "Graveyard"
          )
            return false;}
```
but this ended up breaking *every* meeting somehow, even tho we only put it in this card

Reply:
this also won't work on passives like dreamer, trickster, granny without changing the engine